src/ruleLoader.ts

```diff
@@ -57,26 +57,40 @@ export function findRule(name: string, rulesDirectory?: string) {
         return Rule;
     }
 
-    // then check for rules within the first level of rulesDirectory
-    if (rulesDirectory) {
-        Rule = loadRule(rulesDirectory, camelizedName);
-        if (Rule) {
-            return Rule;
+    // normalize the rulesDirectories argument
```

Comment:
Once again, I did that only for backwards compatibility, since other developers maybe were using this method passing a string.